frontend/src/Components/AlgorithmPage/AlgorithmPage.js

```diff
@@ -51,6 +58,7 @@ class AlgorithmPage extends Component {
                                         title={ this.props.title }
                                         visualizeResults={ this.visualizeResults.bind(this) }
                                         setChartTitles={ this.setChartTitles.bind(this) }
+                                        checkCBA={ this.checkCBA.bind(this) }
```

Comment:
Quick tip, if you write the `checkCBA` function in ES6 arrow syntax, you don't have to `bind`, it implicitly does it for you. No need to change anything though.

Reply:
Thanks for the tip! Is the arrow function the preferred method/better practice?

Reply:
The arrow function is my preference and it seems that it's the direction the community is going.